stateMatrix.m

```diff
@@ -3,9 +3,11 @@
 global TaskParameters
 
 %% Define ports
-LeftPort = floor(mod(TaskParameters.GUI.Ports_LMRAir/1000,10));
-CenterPort = floor(mod(TaskParameters.GUI.Ports_LMRAir/100,10));
-RightPort = floor(mod(TaskParameters.GUI.Ports_LMRAir/10,10));
+LeftPort = floor(mod(TaskParameters.GUI.Ports_LMRAudLRAir/100000,10));         
+CenterPort = floor(mod(TaskParameters.GUI.Ports_LMRAudLRAir/10000,10));
+RightPort = floor(mod(TaskParameters.GUI.Ports_LMRAudLRAir/1000,10));
+LeftSound = floor(mod(TaskParameters.GUI.Ports_LMRAudLRAir/100,10));
+RightSound = floor(mod(TaskParameters.GUI.Ports_LMRAudLRAir/10,10));
```

Comment:
I'd suggest renaming `LeftSound` and `RightSound` to `LeftSoundPort` and `RightSoundPort` to be mroe specific about the variable holds

Reply:
agree! solved